frontend/src/index.js

```diff
@@ -5,23 +5,43 @@ import { Provider } from 'react-redux'
 import { resetContext, getContext } from 'kea'
 import { routerPlugin } from 'kea-router'
 import { loadersPlugin } from 'kea-loaders'
+import NProgress from 'nprogress'
 
 import App from './scenes/App'
 import { toast } from 'react-toastify'
 
+let loadingCounter = 0
+let loadingTimeout
+
+const startLoading = () => {
+    loadingTimeout = window.setTimeout(() => NProgress.start(), 500)
```

Comment:
With a big dashboard (like on app) that takes a while to load, this somehow makes the dashboard feel slower rather than faster, as you're waiting for it to finish before doing anything.

I think it's fine everywhere else

Reply:
Another thing: If loading takes a long time on one screen and you navigate away, it will carry on showing the loader

Reply:
There are too many edge cases to make this work fine: so I just removed this nprogress for now.